spec/ic.did

```diff
@@ -29,6 +29,10 @@ type http_request_error = variant {
   timeout;
   bad_tls;
   invalid_url;
+  transform_error;
+  dns_error;
+  unreachable;
```

Comment:
Question: what does `unreachable` mean here?

Reply:
That the remote server was unreachable